var/spack/repos/builtin/packages/deconwolf/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Deconwolf(CMakePackage):
+    """deconwolf is a software for 3-D deconvolution of fluorescent wide-field images."""
+
+    homepage = "https://deconwolf.fht.org/"
+    git = "https://github.com/elgw/deconwolf.git"
+    url = "https://github.com/elgw/deconwolf/archive/refs/tags/v0.4.5.tar.gz"
```

Comment:
```suggestion
    homepage = "https://deconwolf.fht.org/"
    url = "https://github.com/elgw/deconwolf/archive/refs/tags/v0.4.5.tar.gz"
    git = "https://github.com/elgw/deconwolf.git"
```

It's standard convention to put the git property after the url.